src/protagonist/API/Features/DeliveryChannels/Requests/GetDeliveryChannelPolicyCollections.cs

```diff
@@ -0,0 +1,62 @@
+﻿using DLCS.Model.Assets;
+using DLCS.Model.Policies;
+using DLCS.Repository;
+using Hydra.Collections;
+using MediatR;
+
+namespace API.Features.DeliveryChannels.Requests;
+
+public class GetDeliveryChannelPolicyCollections: IRequest<HydraCollection<HydraNestedCollection<DeliveryChannelPolicy>>>
+{
+    public int CustomerId { get; }
+    public string BaseUrl { get; }
+    public string JsonLdId { get; }
+    
+    public GetDeliveryChannelPolicyCollections(int customerId, string baseUrl, string jsonLdId)
```

Comment:
Feels wrong to be passing `BaseUrl` and `JsonLdId` values down to the request here. 

All other API endpoints construct the hydra model in the controller using the values sent back from the mediatr handler. Whereas this builds the entire hydra object in the mediatr handler. Hydra concerns shouldn't go deeper than the controller.

It might be that this specific request doesn't suit construction in Mediatr as it's all Hydra building, or the mediatr handler returns just enough information for the object to be constructed.